hazelcast-jet-all/src/main/java/com/hazelcast/jet/server/JetCommandLine.java

```diff
@@ -1128,10 +1128,15 @@ private static void printRow(SqlRow row, int[] colWidths, Alignment[] alignments
         builder.append("|");
         int columnCount = row.getMetadata().getColumnCount();
         for (int i = 0; i < columnCount; i++) {
-            String colValue = row.getObject(i).toString();
-            colValue = sanitize(colValue, colWidths[i]);
+            String colString;
+            Object columnValue = row.getObject(i);
+            if (columnValue != null) {
+                colString = sanitize(columnValue.toString(), colWidths[i]);
+            } else {
+                colString = "NULL";
+            }
```

Comment:
Equivalent, but shorter code:
```suggestion
            String colString = sanitize(String.valueOf(row.<Object>getObject(i)), colWidths[i]);
```

Reply:
Won't `String.valueOf` return a lowercase `null`, though?

Reply:
Yes, it will:
![image](https://user-images.githubusercontent.com/9109877/103638251-f31ffa00-4f4c-11eb-981a-eb50b910e236.png)